src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -59,6 +64,38 @@ public void initializeAuthedSpeechlet() throws URISyntaxException {
     }
 
     @Test
+    public void getStopDetails() throws SpeechletException, URISyntaxException {
```

Comment:
Cool, thanks for improving code coverage. :smile: 
I pushed a commit to your branch that fixes the unit test.